app/src/main/java/com/example/nockanakalinowej/Model/GameController.java

```diff
@@ -35,9 +53,9 @@
 
     public GameController() {
         levels = new Level[GameProperties.levelsNo];
-
+        Random rand = new Random();
         for (int i = 0; i < GameProperties.levelsNo; i++) {
-            Random rand = new Random();
+
 
             levels[i] = new Level(i, GameProperties.levelsProperties[i]);
             levels[i].setVariant(abs(rand.nextInt()) % 2 + 1);

```

Comment:
change to:
levels[i].setVariant(abs(rand.nextInt()) % GameProperties.viariantsNo + 1);